backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -40,24 +40,27 @@ def process_row_inner(self, sample_container, measures, process, process_measure
         volume_used = process_measurement['volume_used']
         sample_volume = source_sample_obj.volume
         final_volume = None
-        
+
         if initial_volume is None:
-            self.errors['initial_volume'] = 'Initial volume must be specified'
+            if measured_volume is not None:
+                self.errors['initial_volume'].append('Current Volume must be specified if Measured Volume is specified')
```

Comment:
initial_volume => current_volume as error key to match the column name in the template.